nodes/cma/nodes-cma-autoscaling-custom-rn.adoc

```diff
@@ -60,7 +75,9 @@ Before installing this version of the Custom Metrics Autoscaler Operator, remove
 [id="nodes-pods-autoscaling-custom-rn-2112-332-bugs_{context}"]
 === Bug fixes
 
-* Because the Custom Metrics Autoscaler Operator version 3.11.2-311 was released without a required volume mount in the Operator deployment, the Custom Metrics Autoscaler Operator pod would restart every 15 minutes. This fix adds the required volume mount to the operator deployment. As a result, the Operator no longer restarts every 15 minutes.  (link:https://issues.redhat.com/browse/OCPBUGS-22361[*OCPBUGS-22361*])
+* Because the Custom Metrics Autoscaler Operator version 3.11.2-311 was released without a required volume mount in the Operator deployment, the Custom Metrics Autoscaler Operator pod would restart every 15 minutes. This fix adds the required volume mount to the operator deployment. As a result, the Operator no longer restarts every 15 minutes. (link:https://issues.redhat.com/browse/OCPBUGS-22361[*OCPBUGS-22361*])
```

Comment:
```suggestion
* Because the Custom Metrics Autoscaler Operator version 3.11.2-311 was released without a required volume mount in the Operator deployment, the Custom Metrics Autoscaler Operator pod would restart every 15 minutes. This fix adds the required volume mount to the Operator deployment. As a result, the Operator no longer restarts every 15 minutes. (link:https://issues.redhat.com/browse/OCPBUGS-22361[*OCPBUGS-22361*])
```